Add detekt run configuration for format

att55 opened the pull request:
## Issue

- https://github.com/DroidKaigi/conference-app-2023/issues/1034

## Overview (Required)

~~I add Spotless run configuration for format.~~

~~As I had asked before, I reinstalled Spotless.~~
~~https://github.com/DroidKaigi/conference-app-2023/issues/1034#issuecomment-1703854345~~

~~Is what I did correct?~~

I add a configuration to run detekt.

## Screenshot (Optional if screenshot test is present or unrelated to UI)

<img width="292" alt="スクリーンショット 2023-09-12 0 10 57" src="https://github.com/DroidKaigi/conference-app-2023/assets/25548003/2617cd10-2b5b-4cb5-a3d5-4a1fb9713cb3">

takahirom commented:
We are using detekt like this. 
`./gradlew detekt --auto-correct`
It uses ktlint internally. So we don't have to introduce spotless. 👀

att55 commented:
@takahirom 
Thank you for your reply!
So, I should have added a configuration file for detekt.

I removed spotless and added a configuration file to run for detekt.
Please check. 🙏🏼 

<img width="292" alt="スクリーンショット 2023-09-12 0 10 57" src="https://github.com/DroidKaigi/conference-app-2023/assets/25548003/09559215-c57f-4c48-acda-807389b0565e">

takahirom reviewed (approved):
Looks great! Thanks for adding new mechanism!
